indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java

```diff
@@ -324,6 +326,7 @@ public String getVersion(final Interval interval)
     this.plumber = plumberSchool.findPlumber(dataSchema, tuningConfig, metrics);
 
     Supplier<Committer> committerSupplier = null;
+    final File firehoseTempDir = Files.createTempDir();
```

Comment:
This should be inside the task working dir -- a subdirectory of `toolbox.getTaskWorkDir()` -- not system temp directory.